module-services/service-time/AlarmOperations.cpp

```diff
@@ -338,6 +333,26 @@ namespace alarms
         }
     }
 
+    bool AlarmOperationsCommon::turnOffAlarmIfFoundInOngoingEvents(std::uint32_t id)
```

Comment:
`const`

Reply:
If function parameters is not a redirection(reference or pointer) so parameters is copied or moved I do prefer do not use const

Reply:
It's true in the case of headers. For implementation, I can't see a reason not to use const.